server/routes/api.js

```diff
@@ -1007,33 +1007,137 @@ router.post('/capcodeRefresh', isLoggedIn, function(req, res, next) {
   })
 });
 
-router.post('/capcodeExport', isLoggedIn, function(req, res, next) {
+router.post('/capcodeExport', isLoggedIn, function (req, res, next) {
   nconf.load();
   var dbtype = nconf.get('database:type');
   var filename = 'export.csv'
   db.from('capcodes')
     .select('*')
-    .modify(function(queryBuilder) {
+    .modify(function (queryBuilder) {
       if (dbtype == 'oracledb')
         queryBuilder.orderByRaw(`REPLACE("address", '_', '%')`);
       else
         queryBuilder.orderByRaw(`REPLACE(address, '_', '%')`)
     })
     .then((rows) => {
-      converter.json2csv(rows, function(err, data) {
+      converter.json2csv(rows, function (err, data) {
         if (err) {
           res.status(500).send(err);
         } else {
           res.status(200).send({ 'status': 'ok', 'data': data })
         }
-      }) 
+      })
     })
     .catch((err) => {
       logger.main.error(err);
       return next(err);
     })
 });
 
+router.post('/capcodeImport', isLoggedIn, function (req, res, next) {
+  for (var key in req.body) {
+    //remove newline chars from dataset
+    req.body[key] = req.body[key].replace(/\r/g, '');
```

Comment:
Could here be: `req.body[key] = req.body[key].replace(/[\r\n]/g, '');` ?

Reply:
> req.body[key] = req.body[key].replace(/[\r\n]/g, '');

Does this replace both \r and \n individually?

Reply:
yes, as far as I remember it does (`[]` is for specifying a group of chars that match in regex), but better to test it to be sure